uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/LeaveRequest.php

```diff
@@ -130,6 +128,28 @@ protected function update($params) {
     return $this->createRequestWithBalanceChanges($params);
   }
 
+  /**
+   * Return an error message either for specific invalid status transition cases
+   * or a default generic message
+   *
+   * @param array $params
+   *
+   * @return string
+   */
+  private function getErrorMessageForInvalidStatusTransition($params) {
```

Comment:
Can you rename this test to reflect that it is to test the error message when transition is to an Approved status